Automattic/wp-calypso
# Checkout: Add Pix payment method

## Proposed Changes

This adds a new payment method, Pix, to the list of payment methods available in checkout. Like other payment methods, this will only be available if the shopping cart allows it, so it is not currently enabled.

Payment form             |  Confirmation modal
:-------------------------:|:-------------------------:
<img width="582" alt="Screenshot 2024-01-30 at 6 06 59 PM" src="https://github.com/Automattic/wp-calypso/assets/2036909/09af6e08-a11e-4258-9249-ae74995e18b2"> | <img width="1273" alt="Screenshot 2024-02-01 at 3 34 52 PM" src="https://github.com/Automattic/wp-calypso/assets/2036909/f2f754e8-c132-4b08-a176-20d4840fff89">

~Requires D130371-code~ (That has been merged)

## Testing Instructions

- To use Pix as a payment method option in checkout, first you'll need to have BRL as your currency, and then you'll need to select "Brazil" as the country inside checkout (and a Brazilian postal code like `61919-230`).
- Use Ebanx [test customer info](https://developer.ebanx.com/docs/resources/testCustomerData/) to fill out the Pix form. "CEP" means "Postal code" and "CPF" means "Taxpayer Identification Number".
- Click to submit the payment.
- Use the Ebanx dashboard (you must be an Ebanx account admin) to confirm the payment.
- Verify that the purchase completes successfully.
